diff --git a/third_party/thrift/thrift/transport/TTransport.h b/third_party/thrift/thrift/transport/TTransport.h
index 8d5d86b16a..f2d15ca2cb 100755
--- a/third_party/thrift/thrift/transport/TTransport.h
+++ b/third_party/thrift/thrift/transport/TTransport.h
@@ -22,6 +22,7 @@
 
 #include "thrift/Thrift.h"
 #include "thrift/transport/TTransportException.h"
+#include <cstdint>
 #include <memory>
 #include <string>
 
